download_test_data.sh

```diff
@@ -24,6 +24,7 @@ wget \
 	-R "*.html*,*.gif" \
 	--no-host-directories --cut-dirs=1 \
 	--no-parent \
+	--level=inf \
```

Comment:
what is the purpose of `--level=inf`?

Reply:
wget has a default limit of 5 for recursing into directories. With the new data tree, we have a deeper structure than that